docs/pages/admin-guides/teleport-policy/integrations/aws-sync.mdx

```diff
@@ -95,16 +94,16 @@ it's possible to reuse it as long as the following requirements are met:
 
 - On step 2, you match the `discovery_group` with the existing Discovery Service's
 `discovery_group`.
-- Access Graph service is reachable from the machine where Discovery Service runs.
+- Graph Explorer service is reachable from the machine where Discovery Service runs.
```

Comment:
Most of this is good, but I would note again that we run the access graph service for this.